girder_worker/tasks.py

```diff
@@ -1,25 +1,20 @@
 import core
-from .core.utils import JobManager, JobStatus
+from girder_worker.utils import JobStatus
 from .app import app
 
 
 def _cleanup(*args, **kwargs):
     core.events.trigger('cleanup')
 
 
-@app.task(name='girder_worker.run', after_return=_cleanup)
-def run(*pargs, **kwargs):
-    jobInfo = kwargs.pop('jobInfo', {})
-    retval = 0
+@app.task(name='girder_worker.run', bind=True, after_return=_cleanup)
+def run(task, *pargs, **kwargs):
+    kwargs['_job_manager'] = task.job_manager \
+        if hasattr(task, 'job_manager') else None
 
-    with JobManager(logPrint=jobInfo.get('logPrint', True),
```

Comment:
The docstring in JobManager still says it's a context manager. We should either change the docstring or add the `__enter__` and `__exit__` methods back in so it can be used as such.